app/components/cms_text_with_asides_component/cms_text_with_asides_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render GovGridRowComponent.new do |row| %>
+  <%= row.with_column("two-thirds") do %>
+    <%= render CmsRichTextBlockComponent.new(blocks: @blocks, with_wrapper: false) %>
+  <% end %>
+
+  <%= row.with_column("one-third") do %>
+
+    <% @aside_models.each do |aside| %>
+      <% if aside[:model] %>
+        <%= render aside[:model].render %>
+      <% else %>
+        <!-- missing aside <%= aside[:slug] %> -->
```

Comment:
Does this comment need removing?

Reply:
Good question, I had it on the references to the hard coded asides we had in the dashboard, but I guess in the strapi version they should never be missing. I will remove it.